pythonExecSource/docs/Usage.md

```diff
@@ -0,0 +1,302 @@
+   - At this point, the source is *connected* and operational.
+ - Once connected, the source can be the target of SELECT statements.
+
+The sequence above requires that the connector be told of the Vantiq installation to which to connect, appropriate credentials for doing so, and the name of the source it to perform the `connectExtension` operation.
+
+This information should be provided in a configuration file. 
+(Please read the [Connector SDK's server config documentation](../../extpsdk/README.md#serverConfig) first.)
+Specifically, create a file named `server.config` on a `serverConfig` directory within the working directory in which the connector run.
+
+The information required is placed in that file as follows:
+
+```
+targetServer = ...
+authToken = ...
+source = ...
+```
+
+An example file might be
+
+```
+targetServer = https://dev.vantiq.com
+authToken = _cDWBfZLNO9FkXd-twjwKnVIBZSGwns35nF4nQFV_ps=
+source = pythonSource
+```
+
+For users who may not want to write the `authToken` property to a file because of its sensitive nature, set the environment variable `CONNECTOR_AUTH_TOKEN` to its value. If the `authToken` is specified in the `server.config` document, that value will take precedence.
+Otherwise, if the `authToken` is not set in the configuration file, the system will retrieve whatever value is provided in the environment variable.
+
+> Note that this token will not work -- you will need to create your own
+> within a Vantiq installation
+
+You should also provide an approriate `logger.ini` file in the same directory.
+An example one is provided at `src/test/resources/logger.ini` in this project.
+
+#### Creating a Docker Image for your Connector
+
+The general process for creating a docker image is described in the Extension Sources Overview.
+
+However, except for the most trival cases, you will generally want to include other Python packages in the installation within which the connector runs.  If, for example, you are using some machine learning functions in the code to run, you will want to have the appropriate packages installed in the connector's Python environment.
+
+In the case of a Docker image, each such image is its own environment.  Consequently, you must provide for the installations of the Python packages for each Docker image created.
+
+To do this, we make use of the `connectorSpecificInclusions` property to perform the build. The `connectorSpecificInclusions` identifies a directory that contains files that are to be included in the connector's Docker image.  For the this connector, we require that that property be specified, and that the directory specified contain a file named `localRequirements.txt`.  This should be a file in the [pip requirements format](https://pip.pypa.io/en/stable/reference/requirements-file-format/).  This file will be provided to pip as part of the Docker image build to load the Python packages required by this connector.
```

Comment:
` For the this connector, we require that that property be specified, and that the directory specified contain a file ...` ->  `For this, we require that the property be specified, and that the directory specified contains a file ...`